content/metrics/metrics-10/contents.lr

```diff
@@ -0,0 +1,16 @@
+title: Why do the graphs end 2 days in the past and not today?
+---
+key: 10
+---
+description:
+
+Relays and bridges report some of the data in 24-hour intervals which
+may end at any time of the day.  And after such an interval is over relays
+and bridges might take another 18 hours to report the data.  We cut off
+the last two days from the graphs, because we want to avoid that the last
+data point in a graph indicates a recent trend change which is in fact
+just an artifact of the algorithm.
+---
+seo_slug: metrics
```

Comment:
1. It would be nice to have a short URL, for example, graphs.. So you will need to rename this question directory.
2. Change the seo_slug and _slug.